content/documentation/pages/6-recipes/1-polyglot/2-polyglot-python-task.md

```diff
@@ -6,31 +6,62 @@ description: 'Python/Docker as SCDF Task'
 
 # Python/Docker as SCDF Task
 
-This example illustrates how run an Python script as [Data Flow Task](http://docs.spring.io/spring-cloud-dataflow/docs/2.1.0.BUILD-SNAPSHOT/reference/htmlsingle/#spring-cloud-dataflow-task).
+This recipe shows how to run a custom Python script as a [Data Flow Task](http://docs.spring.io/spring-cloud-dataflow/docs/%scdf-version-latest%/reference/htmlsingle/#spring-cloud-dataflow-task) and how to orchestrate later as [Composed Tasks](http://docs.spring.io/spring-cloud-dataflow/docs/%scdf-version-latest%/reference/htmlsingle/#spring-cloud-dataflow-composed-tasks).
 
-**TODO: Describe what the Task is doing**
+The approach requires the scripts to be bundled as docker images and is applicable for the `Local` and `Kubernetes` runtime platform.
 
-A small helper class [TaskStatus](https://github.com/tzolov/scdf-polyglot-experiments/blob/master/python_task_with_status/util/task_status.py) is used to write task state to the Task table created by Data Flow.
+Following diagram helps to visualize the component involved and the control flow of this approach.
 
-The source code can be found [here](https://github.com/tzolov/scdf-polyglot-experiments/tree/master/python_task_with_status)
+![SCDF Python Tasks](images/python_docker_task_self_managed_status.png)
 
-**TODO provide a .zip file of all the source code**
-**TODO: Source code to be moved into data flow samples repo in a directory structure that mimics the dataflow.io directory structure. There is already some content in there to get an idea of the directory structure.**
+When launched, the custom script executes its task and completes with either a successfully or failure status.
+Because this is not a standard Sprint Cloud Task, the script is responsible to manage its life cycle state within the Data Flow database!
```

Comment:
> Because this is not a standard Sprint Cloud Task application, it is the user's responsibility to manage the life cycle and update the state to the shared database that is also used by SCDF.